server/modules/record/repository/recordRepository.js

```diff
@@ -118,6 +118,22 @@ const fetchRecordUuids = async (surveyId, client = db) => await client.map(
   R.prop('uuid')
 )
 
+const fetchRecordCreatedCountsByDates = async (surveyId, from, to, client = db) => await client.any(`
+    SELECT
+      COUNT(r.uuid),
+      to_char(date_trunc('day', r.date_created), 'YYYY-MM-DD') AS date
+    FROM
+      ${getSurveyDBSchema(surveyId)}.record r
+    WHERE
+      r.date_created BETWEEN $1::DATE AND $2::DATE + INTERVAL '1 day'
```

Comment:
should you exclude "preview" records from this count?

Reply:
Yes, just pushed the fix